### PR TITLE
Feature 115254 fix flexberry file

### PR DESCRIPTION
**Pull request by Nasutya**
Renamed property for input in flexberry-file.

### CODE REVIEW

File: addon/components/flexberry-file.js [hunk 1]
```diff
@@ -308,18 +308,18 @@ export default FlexberryBaseComponent.extend({
     Component's input additional CSS-class names.
     See [Semantic UI inputs classes](http://semantic-ui.com/elements/input.html).
 
-    @property class
+    @property inputClass
     @type String
-    @default 'fluid'
+    @default ''
     @example
```
Comment (yzaripova) [inline addon/components/flexberry-file.js]: "Your code sample should be indented beneath the @example tag" see yuidoc.

Comment (yzaripova) [inline addon/components/flexberry-file.js]: Remove this style:
".flexberry-lookup,
  .ui.input > input {
    width: 100%;
  }"

Comment (Nasutya) [inline addon/components/flexberry-file.js]: These fixes are outside the scope of the current task.

**Comment by SeregaMatin** [comment]
PR must be done to release0.5.0 branch, please, send a new one.
